<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>135</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tambah Satuan Jumlah Barang</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Project katalog/data/ui/logo.qrc">
    <normaloff>:/newPrefix/logo.png</normaloff>:/newPrefix/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="editLine">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;


padding: 0 1px;


selection-background-color: darkgray;</string>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="CustomComboBox" name="role">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {border: 2px solid gray;
border-radius:6px;
border-bottom-left-radius:0px;
border-top-left-radius:0px;
padding: 0 5px;

selection-background-color: darkgray; 
border-left:0px}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: gray;
    border-left-style: solid; /* just a single line */
  	border-top-right-radius: 6px; /* same radius as the QComboBox */
    border-bottom-right-radius: 6px;
}

QComboBox::down-arrow {
	image: url(:/newPrefix/drop-down-arrow.png);
	width: 13px;
    height: 13px;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 3px solid darkgray;
    selection-background-color: gray;
}</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="maxCount">
          <number>2147483647</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="placeholderText">
          <string>WOIIII</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;

padding: 0 0px;


selection-background-color: darkgray;</string>
         </property>
         <property name="text">
          <string>Tambah Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>125</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid gray;
border-left : 0px;
padding: 0 0px;

selection-background-color: darkgray;</string>
         </property>
         <property name="text">
          <string>Kurang Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="apply">
       <property name="minimumSize">
        <size>
         <width>251</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>251</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid gray;

padding: 0 0px;
selection-background-color: darkgray;</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomComboBox</class>
   <extends>QComboBox</extends>
   <header>MyWidgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editLine</tabstop>
  <tabstop>role</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>minButton</tabstop>
  <tabstop>apply</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Project katalog/data/ui/logo.qrc"/>
 </resources>
 <connections/>
</ui>
